### PR TITLE
[4.0] Template styles column preview

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
Separate columns for module-position-preview and title in templates styles,
Hide the column "preview" in admin mode, as the feature is made only for frontend templates.

### Testing Instructions
Before patch: 
template-modules-preview

After patch:
template-modules-preview-2

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on baf12a5d39003213a456efa3594e8bceabc60677<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24440">issues.joomla.org/tracker/joomla-cms/24440</a>.</sub>

File: administrator/components/com_templates/tmpl/styles/default.php [hunk 1]
```diff
@@ -36,9 +36,12 @@
 						</caption>
 						<thead>
 							<tr>
-								<td style="width:1%" class="text-center">
+								<th style="width:1%" class="text-center">
```
Comment (brianteeman) [inline administrator/components/com_templates/tmpl/styles/default.php]: Nothing has changed to make this a heading

File: administrator/components/com_templates/tmpl/styles/default.php [hunk 2]
```diff
@@ -36,9 +36,12 @@
...
 									&#160;
-								</td>
+								</th>
+								<th  style="width:5%" class="text-center">
```
Comment (brianteeman) [inline administrator/components/com_templates/tmpl/styles/default.php]: Please add the scope

Comment (chmst) [reply administrator/components/com_templates/tmpl/styles/default.php]: All right.What do you think about swaping columns title and preview? In general we have the title first, then the links. i.e. users with link to permissions.

Comment (brianteeman) [reply administrator/components/com_templates/tmpl/styles/default.php]: yes its always best to frontload the title

**Comment by chmst** [comment]
We also could hide the column "preview" for admin templates, as the feature is only used for frontend templates.

**Comment by brianteeman** [comment]
yes we could. I also thought about disabling the column if the preview is not enabled - but then I will get shouted at for making a hidden feature

**Comment by chmst** [comment]
We could hide it in the admin mode and show it always in the frontend mode. Then it is the same as for column "pages".

**Comment by chmst** [comment]
@ChristineWk, @brianteeman could you re-test please?

File: administrator/components/com_templates/tmpl/styles/default.php [hunk 3]
```diff
@@ -42,6 +42,11 @@
 								<th scope="col">
 									<?php echo JHtml::_('searchtools.sort', 'COM_TEMPLATES_HEADING_STYLE', 'a.title', $listDirn, $listOrder); ?>
 								</th>
+								<?php if ($clientId === 0) : ?>
+									<th  scope="col" style="width:5%" class="text-center">
```
Comment (brianteeman) [inline administrator/components/com_templates/tmpl/styles/default.php]:
```suggestion
									<th scope="col" style="width:5%" class="text-center">
```
Remove extra space

**Comment by brianteeman** [comment]
One small CS issue otherwise its great and I will mark successful when that is fixed

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on d2dd535c53c4d02edd97e6cf631b0226c5b9d3b5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24440">issues.joomla.org/tracker/joomla-cms/24440</a>.</sub>

**Comment by chmst** [comment]
thank you - you fixed it ;)

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on d2dd535c53c4d02edd97e6cf631b0226c5b9d3b5<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24440">issues.joomla.org/tracker/joomla-cms/24440</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24440">issues.joomla.org/tracker/joomla-cms/24440</a>.</sub>

**Comment by wilsonge** [comment]
@zero-24 @SniperSister RIPS please

**Comment by zero-24** [comment]
@wilsonge It seams rips / drone is happy?

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**